lib/ueberauth/strategy/google/oauth.ex

```diff
@@ -27,7 +27,10 @@ defmodule Ueberauth.Strategy.Google.OAuth do
   def client(opts \\ []) do
     config = Application.get_env(:ueberauth, __MODULE__, [])
     opts = @defaults |> Keyword.merge(opts) |> Keyword.merge(config) |> resolve_values()
+    json_library = Application.get_env(:ueberauth, :json_library)
```

Comment:
@snewcomer 

shouldn't that be 

```elixir
json_library = Ueberauth.json_library()
```

?

Reply:
glad somebody looked at my code 😆

Reply:
@snewcomer 😆